src/main/java/connectify/logic/commands/AddPersonCommand.java

```diff
@@ -33,21 +38,24 @@ public class AddPersonCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "owesMoney "
+            + PREFIX_COMPANY + "1";
```

Comment:
Okay i see that we are assigning people to company based on company index. This works!